pages/documentation/guidance/performance/why.md

```diff
@@ -17,7 +17,7 @@ Through USWDS team’s research around performance, we found government teams th
 
 Conversion is the rate at which customers of a website product take a desired action. The desired action depends on the website's goals and purpose. The conversion action for an e-commerce site is often a user completing a purchase, but public sector sites that don't sell anything can also use the idea of conversion. For example, the IRS site's conversion action could be when somebody successfully submits their tax returns; or the Federal Election Commission's conversion action could be when somebody successfully finds data they were looking for.
 
-Performance has been known to be an influencing factor when it comes to the conversion rate for a site. Regardless of the user interface and its purpose, multiple studies have shown that improving performance consistently improves conversion rates over a wide range of applications. According to [surveys done by Akamai](https://www.akamai.com/us/en/about/news/press/2009-press/akamai-reveals-2-seconds-as-the-new-threshold-of-acceptability-for-ecommerce-web-page-response-times.jsp), nearly half of web users expect a site to load in two seconds or less, and they tend to abandon a site that isn’t loaded within three seconds. This means that even a relatively fast site may be losing conversion due to speed. There are real-world examples of how performance can affect conversion, such as:
```

Comment:
I found references to this study but the style itself seems to have been pulled.

I believe it was published in 2009 - is it still relevant?

Reply:
A quick search doesn't give any alternate up-to-date  source. Maybe we just remove the Akamai specific parts?

Reply:
I removed the link to the Akamai press release and its related stats because the source is no longer available. [4f7cd46](https://github.com/uswds/uswds-site/pull/1735/commits/4f7cd462337b5c770781d4ec8c1d81656556bd4a)